custom/icds_reports/management/commands/sql_scripts/urgent_pmo_datapull.sql

```diff
@@ -0,0 +1,85 @@
+ /*
+ Finalize Aggregate  (cost=19316.33..19316.34 rows=1 width=16)
+   ->  Gather  (cost=19315.90..19316.31 rows=4 width=16)
+         Workers Planned: 4
+         ->  Partial Aggregate  (cost=18315.90..18315.91 rows=1 width=16)
+               ->  Parallel Append  (cost=0.00..17852.25 rows=92730 width=2)
+                     ->  Parallel Seq Scan on icds_db_aww_incentive_8048fca9e4eef64030deaa32  (cost=0.00..2787.88 rows=32710 width=2)
+                           Filter: (month = '2019-03-01'::date)
+                     ->  Parallel Seq Scan on icds_db_aww_incentive_44eedad853ea06ce7ff597aa  (cost=0.00..2714.99 rows=35279 width=2)
+                           Filter: (month = '2019-03-01'::date)
+                     ->  Parallel Seq Scan on icds_db_aww_incentive_574bb69e9c1384b0afd7a4bb  (cost=0.00..2571.68 rows=33014 width=2)
+                           Filter: (month = '2019-03-01'::date)
+                     ->  Parallel Seq Scan on icds_db_aww_incentive_50e93a769911a7e32580432d  (cost=0.00..2278.62 rows=29729 width=2)
+                           Filter: (month = '2019-03-01'::date)
+                     ->  Parallel Seq Scan on icds_db_aww_incentive_bcb4a507c13e764811865560  (cost=0.00..1361.54 rows=16363 width=2)
+                           Filter: (month = '2019-03-01'::date)
+                     ->  Parallel Seq Scan on icds_db_aww_incentive_f17ecdf7f4c62b67ccd3ffc6  (cost=0.00..1246.51 rows=15961 width=2)
+                           Filter: (month = '2019-03-01'::date)
+                     ->  Parallel Seq Scan on icds_db_aww_incentive_9328c152ea8a5dd8f402878a  (cost=0.00..1004.10 rows=12648 width=2)
+                           Filter: (month = '2019-03-01'::date)
+                     ->  Parallel Seq Scan on icds_db_aww_incentive_ad462e06e984977ee9a6f93d  (cost=0.00..919.57 rows=10925 width=2)
+                           Filter: (month = '2019-03-01'::date)
+                     ->  Parallel Seq Scan on icds_db_aww_incentive_f097d7fa3f9272bd1db105d5  (cost=0.00..549.27 rows=6582 width=2)
+                           Filter: (month = '2019-03-01'::date)
+                     ->  Parallel Seq Scan on icds_db_aww_incentive_c9fc6833563fe8049903a475  (cost=0.00..543.60 rows=6928 width=2)
+                           Filter: (month = '2019-03-01'::date)
+                     ->  Parallel Seq Scan on icds_db_aww_incentive_44c9a47a653b5bb720a56dcd  (cost=0.00..348.10 rows=4488 width=2)
+ */
+
+select sum(valid_in_month) as child_0_36 from "agg_child_health" inner join awc_location_local on "agg_child_health".state_id = awc_location_local.state_id and "agg_child_health".aggregation_level=awc_location_local.aggregation_level where month='{month}' and age_tranche in ('6', '12', '24', '36') and "agg_child_health".aggregation_level=1;
```

Comment:
hey, we also need to exclude test states in these queries below

Reply:
updated